components/ProtegeProgramCard.tsx

```diff
@@ -0,0 +1,45 @@
+import {Box, Stack, Typography} from '@mui/joy';
+import Card from '@mui/joy/Card';
+import DarkOutlineButton from './DarkOutlineButton';
+import Image from 'next/image';
+
+export default function ProtegeProgramCard({ icon, text, type, description }: any) {
+  return (
+    <Card
+      variant='outlined'
+      className='border-round'
+      sx={{
+        width: {xs: '300px', lg: '350px'},
+        height: 'auto',
+        border: '2px solid #93AED3',
+        backgroundColor: 'white'
+      }}
+    >
+      <Box sx={{
+          position: 'absolute',
+          top: '-50px',
+          left: '50%',
+          transform: 'translateX(-50%)',
+          width: {xs: '90px', sm: '100px'},
+          height: {xs: '90px', sm: '100px'},
+          borderRadius: '50%',
+          border: '2px solid #93AED3',
+          backgroundColor: 'white',
+          display: 'flex',
+          alignItems: 'center',
+          justifyContent: 'center',
+      }}>
+        <Image src={icon} alt={text} width={70} height={70}/>
```

Comment:
we don't need any alt for the images since they don't really hold any important info! just alt="" is fine